charts/jaeger-operator/templates/deployment.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "jaeger-operator.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
It's probably unnecessary here, that's the default behavior, isn't it?

Reply:
In our case we use `helm template` to store the manifests in git. So the namespace is important for generating the manifests offline with `helm template`.
And it would not heart, would it?